bin/install.sh

```diff
@@ -28,12 +28,12 @@ echo -e "Appending following configurations to ~/.zshrc\e[91m"
 
 appendIfNotExist "######## Setup script setup by fantastic-kit ########"
 
-# sourcing fk.sh if it is not already added to the .zshrc file
-appendIfNotExist "source \$HOME/.fantastic-kit/bin/fk.sh"
-
 # export env var for fantastic-kit itself
```

Comment:
This must go first so that all `fk` scripts have access to the variable.

Reply:
Good catch